gatsby-config.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
     author: {
       name: 'Jason Lengstorf',
       minibio: `
-      <strong>Jason Lengstorf</strong> is a lead developer & architect at Gatsby.
+      <strong>Jason Lengstorf</strong> is a top banana and does all sorts of excellent things at Netlify.
```

Comment:
```suggestion
      <strong>Jason Lengstorf</strong> is a principal developer experience engineer at Netlify.
```

Reply:
Apparently I can’t merge that suggestion.

Reply:
Done. (you saw that already, buit I didn't have chance to post a comment when I merged this on my phone yesterday). And congrats again!